platforms/jvm/language-java/src/integTest/groovy/org/gradle/api/tasks/compile/JavaCompileProblemsIntegrationTest.groovy

```diff
@@ -216,24 +266,28 @@ class JavaCompileProblemsIntegrationTest extends AbstractIntegrationSpec impleme
             severity == Severity.ERROR
             fqid == 'compilation:java:compiler-err-expected'
             contextualLabel == '\';\' expected'
+            contextualLabel.split(System.lineSeparator()).length == 1
```

Comment:
Like I said above, these checks are never going to fail without the previous assertion failing first. I wouldn't add this